src/Microsoft.Health.Fhir.Shared.Tests/Samples.cs

```diff
@@ -94,6 +94,12 @@ public static Stream GetDefaultConversionTemplates()
+
+        public static string GetSampleConvertDataResponse() => "{ \"resourceType\": \"Bundle\" }";
+
```

Comment:
nitpicking and senile grumbling: Methods implies what some calculation would be performed inside. For example in methods above we read something. Here it's just return a string. Why we need methods when we can just have `const string`?

Reply:
Thanks for the good suggestion. I will use const string instead :)